### PR TITLE
Add Port (getport.io) workflow templates

### PR DESCRIPTION
**Pull request by MPTG94**
Hi,

I'm a solution architect at [Port](https://getport.io), some of our clients are using codefresh and we wanted to give them an easy way to interact with their developer portal as part of their Codefresh CI process.

I opened a PR with a WorkflowTemplate that can be used as part of the Codefresh CI process to interact with Port.

I'll be happy to make additional changes to make sure the template is up to your standards and can be added to your Argo-hub.

### CODE REVIEW

**Review by eti-codefresh — CHANGES_REQUESTED** [review]

File: workflows/port/versions/1.0.0/docs/entity-upsert.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Inputs/Outputs
+
+### Inputs
+
+- `PORT_CREDENTIALS_SECRET` - name of the secret to get the `CLIENT_ID` and `CLIENT_SECRET` from (default: `port-credentials`)
+- `PORT_CLIENT_ID_KEY` - key in the secret where the base64 encoded `PORT_CLIENT_ID` is stored (default: `PORT_CLIENT_ID`);
+- `PORT_CLIENT_SECRET_KEY` - key in the secret where the base64 encoded `PORT_CLIENT_SECRET` is stored (default `PORT_CLIENT_SECRET`);
+- `BLUEPRINT_IDENTIFIER` - identifier of the blueprint to create an entity of;
+- `ENTITY_IDENTIFIER` - identifier of the new (or existing) entity. Leave empty to get an auto-generated identifier;
+- `ENTITY_TITLE` - Title of the new (or existing) entity;
```
[the remark this answers is not in the document]
Comment (MPTG94) [reply workflows/port/versions/1.0.0/docs/entity-upsert.md]: @eti-codefresh added explanation about default value for each field

File: workflows/port/versions/1.0.0/workflowTemplate.yaml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+          - name: ENTITY_ICON
+            default: ""
+          - name: ENTITY_PROPERTIES
+            default: ""
+          - name: ENTITY_RELATIONS
+            default: ""
+      container:
+        name: main
+        imagePullPolicy: Always
+        image: quay.io/codefreshplugins/argo-hub-workflows-port-versions-1.0.0-images-codefresh-port-entity:main
```
[the remark this answers is not in the document]
Comment (MPTG94) [reply workflows/port/versions/1.0.0/workflowTemplate.yaml]: @eti-codefresh updated

File: workflows/port/versions/1.0.0/workflowTemplate.yaml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+          - name: PORT_CLIENT_ID_KEY
+            default: PORT_CLIENT_ID
+          - name: PORT_CLIENT_SECRET_KEY
+            default: PORT_CLIENT_SECRET
+          - name: BLUEPRINT_IDENTIFIER
+          - name: ENTITY_IDENTIFIER
+      container:
+        name: main
+        imagePullPolicy: Always
+        image: quay.io/codefreshplugins/argo-hub-workflows-port-versions-1.0.0-images-codefresh-port-entity:main
```
[the remark this answers is not in the document]
Comment (MPTG94) [reply workflows/port/versions/1.0.0/workflowTemplate.yaml]: @eti-codefresh updated

**Review by eti-codefresh — APPROVED** [review]

**Review by eti-codefresh — COMMENTED** [review]
@MPTG94 approved

**Comment by MPTG94** [comment]
Hey @eti-codefresh, I added version 1.1.0 to this PR because it is also ready.

You can go ahead and merge the PR if that's ok on your side

**Comment by MPTG94** [comment]
@eti-codefresh @francisco-codefresh checking if there is any update, I do not have permissions to merge this

**Pull request merged**